packages/webcert/src/store/user/userMiddleware.ts

```
@@ -142,8 +146,30 @@ const handleGetUserStatistics: Middleware<Dispatch> = ({ dispatch }: MiddlewareA
   )
 }
 
-const handleGetUserTabsSuccess: Middleware<Dispatch> = ({ dispatch }: MiddlewareAPI) => () => (action: AnyAction): void => {
+const handleGetUserStatisticsStarted: Middleware<Dispatch> = ({ dispatch }: MiddlewareAPI) => () => (): void => {
+  dispatch(updateIsLoadingUserStatistics(true))
+}
+
+const handleGetUserStatisticsSuccess: Middleware<Dispatch> = ({ dispatch }: MiddlewareAPI) => () => (action: AnyAction): void => {
   dispatch(updateUserStatistics(action.payload))
+  dispatch(updateIsLoadingUserStatistics(false))
+}
+
+const handleSetUnit: Middleware<Dispatch> = ({ dispatch }: MiddlewareAPI) => () => (action: AnyAction): void => {
+  dispatch(
+    apiCallBegan({
+      url: `/api/user/unit/${action.payload}`,
+      method: 'POST',
+      onStart: setUnitStarted.type,
+      onSuccess: setUnitSuccess.type,
+      onError: apiSilentGenericError.type,
```

Should this result in a silent error?